cli/azd/cmd/cmd_help.go

```diff
@@ -83,7 +83,7 @@ func getCmdHelpDefaultDescription(cmd *cobra.Command) string {
 func getCmdHelpDefaultUsage(cmd *cobra.Command) string {
 	return fmt.Sprintf("%s\n  %s\n\n",
 		output.WithBold(output.WithUnderline("Usage")),
-		"{{if .Runnable}}{{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}{{.CommandPath}} [command]{{end}}",
+		"{{if .HasAvailableSubCommands}}{{.CommandPath}} [command]{{else}}{{if .Runnable}}{{.UseLine}}{{end}}{{end}}",
```

Comment:
The changes on this PR are making all azd-commands to be runnable (azd will add a default `runE` method for any command which has not neither a `runE` definition or an action description definition.
After that, a command with sub-commands becomes `runnable`, and with previous help-template, both conditions are true `HasAvailableSubCommands` and `.Runnable`, which ends up adding the `.UseLine` + `.CommandPath [command]`.

With this change, instead of doing:
```
if runnable {
}
if hasAvailableSubCommands {
}
```
I changed it to:
```
if hasAvailableSubCommands {
}
else if runnable {
}
```

This maintains previous output for commands with sub-commands, (like azd, pipeline, env, etc), when producing the usage line from the help. Making it more relevant to have sub-commands than being runnable.